lib/schema/schema.graphql

```diff
@@ -0,0 +1,2302 @@
+type GroupMembership implements IGroupMembership {
```

Comment:
I got the updated schema from Chris, who got it from their Apollo UI. This is more of a backend thing so I will defer to them for suggestions.

Reply:
In #38 I have the latest backend schema from `mmdata`, not sure if it's exactly the same as here. I also can't guarantee that it's entirely correct. For userHandle, you're right that it should not be nullable, but it is nullable in our backend at the moment. I'll open a PR to mark that non-nullable and can update in this repo later. 

This schema was generated by running the backend locally and downloading this file from the local [Apollo Server playground](http://127.0.0.1:3000/mmdata/api/graphql), in the Schema/SDL view.

Reply:
FYI it may be possible that we have a "user" that hasn't actually gone through signup (e.g. if we create a "user" for someone once they land on the signup page and accept creation of cookies). I don't know if this is the intended functionality of this collection, but we need to make some decision there prior to modifying this schema to require fields like userHandle, email, etc.